### PR TITLE
Updated note on Tidyverse and added hyperlink

### PR DESCRIPTION
**Pull request by Kenizzer**
Following up on #719 and #748 

Added a small amount of additional text to denote these packages are designed to work together (from https://tidyverse.tidyverse.org/). Also added a hyperlink for the tidyverse website.

### CODE REVIEW

File: _episodes_rmd/13-dplyr.Rmd [hunk 1]
```diff
@@ -50,9 +50,9 @@ even find the `dplyr` grammar easier to read.
 > ## Tip: Tidyverse
 >
 > `dplyr` package belongs to a broader family of opinionated R packages
-> specifically designed for data science called "Tidyverse". Some of these
-> packages will be covered along this course, but you can find more 
-> complete information here: https://www.tidyverse.org/.
+> specifically designed to work harmoinously for data science called the "Tidyverse".
```
Comment (skanwal) [inline _episodes_rmd/13-dplyr.Rmd:53]: is this meant to be "to work harmoniously"

Comment (Kenizzer) [reply _episodes_rmd/13-dplyr.Rmd]: Yes, thank you for pointing out the typo! I fixed the error and made a new commit to the branch.

**Comment by naupaka** [comment]
@Kenizzer thanks for the PR! 

I'd recommend a slight change in wording for clarity, from 

"specifically designed to work harmoniously for data science called the "Tidyverse"

to

"for data science called the "Tidyverse". These packages are specifically designed to work harmoniously together."

**Comment by naupaka** [comment]
Also please note that all the Carpentries repos will be shifting over to a new infrastructure at the end of the month, so any PRs need to be merged and closed before then.

**Comment by matthieu-bruneaux** [comment]
Thank you for the PR @Kenizzer! And thank you for your input @skanwal and @naupaka!

I applied the slight change in wording suggested by @naupaka.

**Pull request merged**